packages/hawtio/src/util/jolokia.ts

```diff
@@ -91,14 +126,14 @@ export function onGenericSuccessAndError<S, E, O extends BaseRequestOptions>(suc
  * The default error handler which logs errors either using debug or log level logging
  * based on the silent setting.
  */
-function defaultErrorHandler(): NonNullable<RequestOptions['error']> {
-  return (response: ErrorResponse) => {
+function defaultErrorHandler(): ErrorCallback {
+  return (response: JolokiaErrorResponse, _index: number) => {
     if (!response.error) {
       return
     }
 
     const req = response.request
-    const operation = req?.type === 'exec' ? req.operation : 'unknown'
+    const operation = req?.type === 'exec' ? 'exec/' + req.operation : req ? req.type : 'unknown'
```

Comment:
is it correct?

Reply:
It's only for logging - I wanted to have actual type when type is not `exec`.